Source/Parallelization/GuardCellManager.cpp

```diff
@@ -251,7 +251,7 @@ guardCellManager::Init (
         ng_FieldSolverG = ng_alloc_EB;
     }
 #ifdef WARPX_DIM_RZ
-    else if (maxwell_solver_id == MaxwellSolverAlgo::Yee) {
+    else if (maxwell_solver_id == MaxwellSolverAlgo::None || maxwell_solver_id == MaxwellSolverAlgo::Yee) {
```

Comment:
I'm not sure if this is necessary, just tying to get the simulation to run the same way as before first.

Reply:
Indeed there are failures if this is not included since the EB logic uses `ng_FieldSolver`.